recipesearch/src/Recipes/SearchResults/SearchResults.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+
+export class SearchResults extends Component {
+  render() {
+    const { searchResults } = this.props;
+    console.log(searchResults)
+    return (
+      <div>
+        {searchResults && searchResults.map((searchResult, index) => (
+          <div key={index}>
+            <h1>{searchResult.strMeal}</h1>
+            <p>{searchResult.strInstructions}</p>
+            <img src={searchResult.strMealThumb} alt="Food pic"/>
```

Comment:
This still doesn't handle cases where the thumbnail image is not provided by the API

Reply:
@edmondatto Let me work on how to fix this

Reply:
I have made figured out that instead of picking a random string of "Food pic", I can allter the alt to indicate Picture of the meal name.
Is that what you were highlighting?_